### ISSUE TITLE
Issue with BFO_0000063

### ISSUE THREAD

**Issue by mcourtot** [OntoZoo/ontobee#96]
Searching for 'precedes' in ontobee:

![screen shot 2016-11-21 at 09 35 27](https://cloud.githubusercontent.com/assets/2214135/20477536/f00e6f86-afcd-11e6-9421-9ed851bf1379.png)

But the link towards BFO_0000063, http://www.ontobee.org/ontology/BFO?iri=http://purl.obolibrary.org/obo/BFO_0000063,  returns a term not found.

cc @cmungall

**Comment by yongqunh** [comment]
It appears this term is hosted in RO:
http://www.ontobee.org/ontology/RO?iri=http://purl.obolibrary.org/obo/BFO_0000063

I think we need to generate a library of special terms like this in order to properly dereference such terms.

Thanks,
Oliver

**Comment by cmungall** [comment]
Ontobee shouldn't hardcode assumptions about URIs.

The fact thathttp://purl.obolibrary.org/obo/BFO_0000063 does not resolve is nothing to do with ontobee, it's a purl.obolibrary issue. We had the correct redirect with OCLC, this was never ported to the new system.

**Comment by yongqunh** [comment]
OK. That’s good if the purl.obolibrary can solve the redirecting issue.
Oliver

**Comment by mcourtot** [comment]
@cmungall could you clarify?

http://purl.obolibrary.org/obo/BFO_0000063 redirects to http://www.ontobee.org/ontology/BFO?iri=http://purl.obolibrary.org/obo/BFO_0000063 which I believe is the right place.

**Comment by cmungall** [comment]
No, this is the right place: http://www.ontobee.org/ontology/RO?iri=http://purl.obolibrary.org/obo/BFO_0000063

**Comment by cmungall** [comment]
close this?

**Comment by mcourtot** [comment]
well it still doesn't resolve so I'd say no ;)

This raises 2 issues:
- Why is a term with a BFO prefix under the RO authority? I know that IDs shouldn't carry semantics (and I had argued for that in the past) but there was strong counter-arguments that the prefix should indicate 'ownership', and therefore when moving terms between resources they should get new IDs.
- How to deal with this from the PURL redirect point of view? This is only for one very specific BFO term - other BFO ones should follow the normal redirect.

**Comment by cmungall** [comment]
https://github.com/oborel/obo-relations/wiki/ROAndBFO

**Comment by cmungall** [comment]
For the redirect you should be able to put something in bfo.yml in the purl github

**Comment by mcourtot** [comment]
This is really confusing - when OBI submitted relations to RO, we updated the prefix and obsoleted the OBI ones to get them RO IDs. Maybe slightly broader scope than the issue currently at hand, but I really think we ought to think about this in a general context and moving terms between ontologies. I agree with your comment above that the prefix should not dictate the master resource, and I thought we did create an AP a little while back specifically for this, though I don't find it in IAO. Something like "under authority of" - maybe we could have a configuration in Ontobee to read those APs and redirect accordingly?
I agree we can do a one off specific redirect, but that doesn't seem very sustainable...

**Comment by yongqunh** [comment]
If needed, Ontobee can configure to make it read fine and redirect right. Let us know if this is allowed, and then we can do it.

**Comment by cmungall** [comment]
rdfs:isDefinedBy?
